package.json

```diff
@@ -24,7 +24,9 @@
     "react-native": "0.72.6",
     "react-native-gesture-handler": "~2.12.0",
     "react-native-safe-area-context": "^4.8.0",
-    "react-native-svg": "13.9.0"
+    "react-native-svg": "13.9.0",
+    "react-native-web": "~0.19.6",
```

Comment:
Why do we need these packages for fixing lint issues?

Reply:
No, we don't need them to fix lint issues. Only added them to ease the testing. If you'd like, I can remove them.

Reply:
Please remove it as it is not part of this PR work.

Reply:
Updated, pls check.

Reply:
There's still an extra package in package.json. There shouldn't be any changes in package.json as part of this work.

Reply:
<img width="381" alt="image" src="https://github.com/aindrila-choudhuri/qur8-front-end/assets/112249538/90207b46-3f7a-4fed-b988-ff8c45dc19a1">
I have changed it and pushed the changes. There are no pending changed from my end.